packages/account/src/tests/mocks/non_implementing_mock.cairo

```diff
@@ -1,7 +1,7 @@
 #[starknet::contract]
 pub(crate) mod NonImplementingMock {
     #[storage]
-    struct Storage {}
+    pub struct Storage {}
```

Comment:
Would it be better if we make empty storage structs not public?

Reply:
I don't think it makes much of a difference, right? But it might be easy to overlook adding the `pub` keyword if we add some field to it. I can remove all `pub` from all empty storage structs if you feel strongly about this.

Reply:
> I don't think it makes much of a difference, right?

Yeahhh this really just occurs with mocks, right? No strong opinions on this. I think it's okay as is